test/test-applications/integrations/Samples.Kafka/Properties/launchSettings.json

```diff
@@ -0,0 +1,21 @@
+{
+  "profiles": {
+    "Samples.Kafka": {
+      "commandName": "Project",
+      "environmentVariables": {
+        "DD_DOTNET_TRACER_HOME": "C:\\repos\\dd-trace-dotnet\\src\\bin\\tracer-home",
```

Comment:
please remove the absolute paths

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/1444/commits/13c2a27450c0accdcb58f61a77ea84f5f859df08